.github/workflows/ci.yml

```diff
@@ -9,18 +9,19 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [18, 20]
```

Comment:
I'm curious as to what benefits using a matrix with 18 and 20 has over just upgrading to 20.

My initial reaction is that I'm not sure using a matrix makes sense here.
* `package-lock.json` is generated using node 20, so expecting `npm ci` to work when using node 18 seems odd.
* No MFEs use a matrix for this step, a specific version is expected
  * Any MFE created using this template would start off with a CI check that needs to pass on both 18 and 20

Reply:
The motivation behind the matrix is to try - if at all possible - to get MFEs and libraries to work with both Node 18 and 20 for a transitional period, much like we did for the Node 16 -> 18 upgrade.  This is so we can have Tutor - while still on Node 18 - work with MFEs that have already been tested to work on 20.  And then when all Tutor-supported MFEs are Node 20-ready, we flip over Tutor, and remove 18 from the Matrix.

But yes, it may be that in some cases it won't be possible.  It seems to be working fine here, though.